### PR TITLE
Updated Spanish translation

### PR DESCRIPTION
**Pull request by DeltaVeintiuno**
Hello, I bring here the updated Spanish translation, I am sorry I couldn't bring it before, I was too busy. As always, I hope it helps, and have a nice day.

### CODE REVIEW

**Comment by fsobolev** [comment]
> I am sorry I couldn't bring it before, I was too busy.

No worries, I understand, we all have a lot of things to do :)

File: po/es.po [hunk 1]
```diff
@@ -37,15 +37,14 @@ msgstr ""
...
 "Time Switch is an easy-to-use program that allows you to power off, reboot "
 "or suspend your system, send a notification (optionally with a sound signal) "
 "or execute any command on timer."
 msgstr ""
-"Time Switch es un simple programa que te permite apagar, reiniciar, "
-"suspender tu sistema, mandar una notificación (opcionalmente con sonido) o "
-"ejecutar un comando con un temporizador."
+"Time Switch es un simple programa que te permite apagar, reiniciar \n"
+"o suspender tu sistema, mandar una notificación (opcionalmente con sonido)\n"
+"o ejecutar cualquier comando con un temporizador."
```
Comment (fsobolev) [inline po/es.po:47]: Newlines (\n) are not needed there, just spaces

**Review by fsobolev — CHANGES_REQUESTED** [review]

File: po/es.po (hunk 1 above)
Comment (DeltaVeintiuno) [reply po/es.po:47]: Alright, it's fixed now

Comment (fsobolev) [reply po/es.po:47]: You forgot spaces

Comment (DeltaVeintiuno) [reply po/es.po:47]: ups, my bad. Now there are.

**Review by fsobolev — APPROVED** [review]

**Comment by fsobolev** [comment]
Thank you! 😊

**Pull request merged**